backend/app/models/llm/request.py

```
@@ -1,24 +1,20 @@
 from typing import Annotated, Any, Literal, Union
 
-from uuid import UUID
+from uuid import UUID, uuid4
 from sqlmodel import Field, SQLModel
 from pydantic import Discriminator, model_validator, HttpUrl
+from datetime import datetime
+from app.core.util import now
 
+import sqlalchemy as sa
+from sqlalchemy.dialects.postgresql import JSONB
+from sqlmodel import Field, SQLModel, Index, text
```

Ruff (0.14.14)

Redefinition of unused `Field` from line 4: `Field` redefined here

Remove definition: `Field`

(F811)